app/presenters/flood_risk_engine/reviewing/row_builders/exemption_row_builder.rb

```diff
@@ -0,0 +1,16 @@
+module FloodRiskEngine
+  module Reviewing
```

Comment:
Yeah I think you are right. Might make them private methods in review presenter - though the class will be pretty big